go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/giantswarm/devctl
 
-go 1.16
+go 1.18
```

Comment:
Why not `go 1.19` here ?

Reply:
Yeah, ain't gonna work :smile: 

```
#!/bin/bash -eo pipefail
go mod tidy && git diff --exit-code

go: go.mod file indicates go 1.19, but maximum version supported by tidy is 1.18

Exited with code exit status 1

CircleCI received exit code 1
```

It can probably work in the next PR, once go is updated to v1.19.x the first time.

Reply:
I assume it needs `architect-orb` to support go v1.19.x, which is the next PR according to the docs.